docs/dist/pre-commit.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+CODE_SNIFFER=$(which phpcs)
+if [[ -z "$CODE_SNIFFER" ]];then
+    if cat /etc/*-release|grep -q '^ID=\(debian\|ubuntu\)';then
+        sudo apt-get install php-codesniffer
+        CODE_SNIFFER=$(which phpcs)
+    fi
+    if [[ -z "$CODE_SNIFFER" ]];then
```

Comment:
Lines 5 ... 8 can be removed. Pre-commit hooks are not expected to ask for a password and to install software. For Travis, the package is now automatically installed anyway.

Some general remark: PalMA used to have very view code comments. We now have achieved an average number (see https://www.openhub.net/p/palma/factoids#FactoidCommentsAverage). I'd like to keep that status. :-)